### PR TITLE
first version for the global interface

### PR DESCRIPTION
**Pull request by antoine-de**
first version just to check if that's ok for you.

needs:
 - [ ] comments 
 - [ ] tests
 - [ ] better handling of external ref if possible

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py [hunk 1]
```diff
@@ -28,19 +28,82 @@
...
 from jormungandr import app
 from jormungandr.interfaces.v1.serializer import api, jsonschema
 import serpy
 
+from jormungandr.interfaces.v1.serializer.base import LiteralField, LambdaField
+
+BASE_PATH = 'v1'
+
+
 class ApisSchema(serpy.Serializer):
```
Comment (ooga) [inline source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py]: Need to be removed

File: source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py [hunk 2]
```diff
@@ -28,19 +28,82 @@
...
+
+
+args_regexp = re.compile(r'<(?P<name>.*?):.*?>')
+
+
+def format_args(rule):
+    formated_rule = args_regexp.sub(lambda m: '{' + m.group('name') + '}', rule)
+    return formated_rule
+
+base_path_regexp = re.compile('^/v1')
```
Comment (ooga) [inline source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py]: shouldn't we use ```BASE_PATH``` ?

**Comment by ooga** [comment]
```
"/coverage/{lon};{lat}/{uri}/physical_modes/{id}": {
    "method": "OPTIONS",
    "$ref": "http://127.0.0.1:5000/v1/coverage/{lon};{lat}/{uri}/physical_modes/{id}"
},
```

The ```$ref``` return a 404

File: source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py (hunk 2 above)
Comment (antoine-de) [reply source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py]: oups you're right

**Pull request merged**